roslaunch_axserver/scripts/roslaunch_server.py

```diff
@@ -69,6 +69,7 @@ def execute_cb(self, gh):
         rospy.loginfo('trigger roslaunch goal ' + gh.get_goal_id().id)
         goal = gh.get_goal()
         command = "exec roslaunch " + goal.pkg + " " + goal.launch_file
+        command+="".join([" "+x+":="+y for (x,y) in zip(goal.parameters,goal.values)])
```

Comment:
The only problem here will be if `y` contains spaces, right? Shall it be better quoted?